playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: adhoc
+  pre_tasks:
+  - name: Print list of known health checks
+    action: openshift_health_check
+    when: checks is undefined or not checks
+  post_tasks:
+  - name: Run health checks
+    action: openshift_health_check
+    args:
+      checks: '{{ checks | default([]) }}'
```

Comment:
Debating whether the `checks` inventory variable should be qualified. Inputs for most other things are expected to be `openshift_<whatever>` usually, so maybe `openshift_checks`? It seems kind of pedantic since this is specific to the playbook. But it's consistent.